### PR TITLE
Major cleanup of urlBuilders

### PR DESCRIPTION
**Pull request by me-andre**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/main/topLevelPages/TopLevelChallenges/ChallengeExplorerContainer.tsx [hunk 1]
```diff
@@ -47,7 +49,7 @@ export const simpleChallengeTagsValueGetter = (c: SimpleChallenge): string[] =>
 
 export const simpleChallengeSpaceDataGetter = (c: SimpleChallenge) => ({
   id: c.spaceId,
-  nameId: c.spaceNameId,
+  // nameId: c.spaceNameId,
```
Comment (ccanos) [inline src/main/topLevelPages/TopLevelChallenges/ChallengeExplorerContainer.tsx:52]: cleanup

File: src/main/topLevelPages/TopLevelChallenges/ChallengeExplorerContainer.tsx [hunk 2]
```diff
@@ -154,6 +155,7 @@ export const ChallengeExplorerContainer: FC<ChallengePageContainerProps> = ({ se
         vision: ch.context?.vision || '',
         matchedTerms: entry.terms,
         member: ch.community?.myMembershipStatus === CommunityMembershipStatus.Member,
+        profile: ch.profile,
       };
     }) || [];
```
Comment (ccanos) [inline src/main/topLevelPages/TopLevelChallenges/ChallengeExplorerContainer.tsx:160]: `??`

File: src/domain/collaboration/activity/ActivityLog/ActivityComponent.tsx [hunk 3]
```diff
@@ -145,7 +114,7 @@ export const ActivityViewChooser = ({ activity, ...rest }: ActivityViewChooserPr
     case ActivityEventType.CalendarEventCreated:
       return <ActivityCalendarEventCreatedView {...activity} {...rest} />;
     case ActivityEventType.UpdateSent:
-      return <ActivityUpdateSentView {...activity} {...rest} />;
+      return <ActivityUpdateSentView {...activity} {...rest} journeyUrl="" />; // TODO provide journeyUrl
```
Comment (ccanos) [inline src/domain/collaboration/activity/ActivityLog/ActivityComponent.tsx:117]: TODO here

File: src/domain/communication/updates/CommunityUpdatesContainer/CommunityUpdatesContainer.tsx [hunk 4]
```diff
@@ -84,11 +82,11 @@ export const CommunityUpdatesContainer: FC<CommunityUpdatesContainerProps> = ({
       }
       const update = await removeUpdate({
         variables: { messageData: { messageID, roomID } },
-        refetchQueries: spaceId && communityId ? [refetchCommunityUpdatesQuery({ communityId })] : [],
+        refetchQueries: communityId ? [refetchCommunityUpdatesQuery({ communityId })] : [],
```
Comment (ccanos) [inline src/domain/communication/updates/CommunityUpdatesContainer/CommunityUpdatesContainer.tsx:85]: What I'm putting here when I have a case like this is `[refetchCommunityUpdatesQuery({ communityId: communityId! })]`

File: src/domain/journey/common/JourneyCard/JourneyCardJoinButton.tsx [hunk 5]
```diff
@@ -7,7 +7,7 @@ import provideStaticProps from '../../../../core/utils/provideStaticProps';
 
 interface JourneyCardJoinButtonProps {
   challengeId?: string;
-  challengeNameId?: string;
+  // challengeNameId?: string;
```
Comment (ccanos) [inline src/domain/journey/common/JourneyCard/JourneyCardJoinButton.tsx:10]: cleanup

File: src/domain/journey/common/tabs/Dashboard/JourneyDashboardView.tsx [hunk 6]
```diff
@@ -105,14 +101,14 @@ const JourneyDashboardView = ({
...
 
-  const journeyLocation: JourneyLocation | undefined =
-    typeof spaceNameId === 'undefined'
-      ? undefined
-      : {
-          spaceNameId,
-          challengeNameId,
-          opportunityNameId,
-        };
+  // const journeyLocation: JourneyLocation | undefined =
```
Comment (ccanos) [inline src/domain/journey/common/tabs/Dashboard/JourneyDashboardView.tsx:104]: cleanup

**Review by ccanos — CHANGES_REQUESTED** [review]
just a few comments to remove

**Review by valentinyanakiev — APPROVED** [review]

**Pull request merged**